Campus-iOS/Map/Cafeterias/MealPlanViewModel.swift

```diff
@@ -21,6 +21,8 @@ final class MealPlanViewModel: ObservableObject {
     init(cafeteria: Cafeteria) {
         self.cafeteria = cafeteria
         self.title = cafeteria.name
+        
+        fetch()
```

Comment:
I would suggest calling `fetch()` not upon the initialization of the ViewModel but something like the `.task {}` modifier :)